cfavml/src/danger/impl_neon.rs

```diff
@@ -279,6 +323,50 @@ impl SimdRegister<f64> for Neon {
     }
 }
 
+impl Hypot<f64> for Neon {
+    #[inline(always)]
+    unsafe fn hypot(x: Self::Register, y: Self::Register) -> Self::Register {
+        // Convert inputs to absolute values
+        let (x, y) = (vabsq_f64(x), vabsq_f64(y));
+
+        // Find the max and min of the two inputs
+        let (hi, lo) = (vmaxq_f64(x, y), vminq_f64(x, y));
```

Comment:
Same vibe as mentioned earlier, best to have these just as two separate lines rather than a tuple.